tests/src/rules/default.js

```diff
@@ -310,7 +234,25 @@ context('TypeScript', function () {
             },
           ],
         }),
-      ],
-    });
-  });
+   ]),
+
+   
+
 });
+
+// #311: import of mismatched case
+if (!CASE_SENSITIVE_FS) {
+  ruleTester.run('default (path case-insensitivity)', rule, {
+    valid: parsers.all([
+      test({
+        code: 'import foo from "./jsx/MyUncoolComponent.jsx"',
+      }),
+     ]),
+    invalid:parsers.all([ 
+      test({
+        code: 'import bar from "./Named-Exports"',
+        errors: ['No default export found in imported module "./Named-Exports".'],
+      }),
+    ]),
+  });
+}
```

Comment:
these can be inlined as well, with something like:
```jsx
CASE_SENSITIVE_FS ? [] : test({
  code: 'import foo from "./jsx/MyUncoolComponent.jsx"',
}),
```
as long as the lists are using concat